components/eventing-controller/testing/natsmock.go

```diff
@@ -0,0 +1,13 @@
+package testing
```

Comment:
I don't think `mock` is the right name here, as we don't really mock the NATS server. What do you think?

Reply:
Sure, changed to Nats